ocean_lib/assets/v4/asset.py

```diff
@@ -31,6 +31,7 @@ def __init__(
         nft: Optional[dict] = None,
         datatokens: Optional[list] = None,
         event: Optional[dict] = None,
+        purgatory: Optional[dict] = None,
```

Comment:
why would you need purgatory? Aquarius shouldn't serve files that are in purgatory, at all. So there's no way they would end up in ocean.py. If there was no purgatory in v3, don't add it in v4, either.

Reply:
I saw in the docs that it was added and wanted to reproduce the whole structure. I will remove it then.